examples/widgets/scale/lv_example_scale_12.c

```diff
@@ -0,0 +1,69 @@
+#include "../../lv_examples.h"
+#if LV_USE_SCALE && LV_BUILD_EXAMPLES
+
+void lv_example_scale_12(void)
```

Comment:
I've just realized that [this example](https://docs.lvgl.io/master/details/widgets/chart.html#axis-ticks-and-labels-with-scrolling) serves the same purpose. 

@vwheeler63 can add a chart example among the scale examples? Maybe just reference it [here](https://github.com/lvgl/lvgl/pull/8225/files#diff-dcee7b13c77037de561f6205b5a4fab99205b1d487553985d08d576bfe6870ffR70)?

If it works, this example can be removed.